app/helpers/storytime_admin/application_helper.rb

```diff
@@ -24,5 +24,37 @@ def respond_to?(method)
         super
       end
     end
+
+    def setup_column_sort(attribute)
+      if params[:sort]
```

Comment:
seems like these conditionals could be simplified quite a bit. The outcomes are always either "asc" or "desc"...we should be able to just have 1 if statement